packages/next-templates/src/app/styling/css/detail-page.css

```diff
@@ -5,7 +5,7 @@ https://www.figma.com/file/gX3fq7k0uUKC45AJbuItDv/Local---Voorbeeld---Bibliothee
   --utrecht-button-group-inline-gap: 0;
 }
 
-.voorbeeld-spotlight-section-heading-style {
+.voorbeeld-utrecht-spotlight-section {
```

Comment:
`utrecht` mag ertussenuit; we gebruiken alleen de voorbeeld-prefix voor custom styling.